test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-1.0
+1.1
```

Comment:
Let us bump up to the major release because a lot of changes went in(something like 2.0)

Reply:
Done